### ISSUE TITLE
Field textarea with buttons

### ISSUE THREAD

**Issue by MathiasGmeiner** [jenstornell/field-engineer#19]
The field `textarea` is only working without buttons.
If I try to use buttons e.g.:

```
type: textarea
buttons:
  - bold
```

I get with `c::set('engineer.debug', true);` an `Undefined index: _level` error and the panel is broken.
With `c::set('engineer.debug', false);` the right side of the panel is just blank.

**Comment by jenstornell** [comment]
Yes, I'm aware of that problem. It's harder to fix than it seems. I wrote a note about it here as well:

https://github.com/jenstornell/field-engineer/blob/master/docs/fields.md

I'll leave this issue open.

**Comment by MathiasGmeiner** [comment]
Thanks for the clarification! I've read the docs, but misread "help buttons".

**Comment by jenstornell** [comment]
@juliangarnier Kirby 3 is around the corner and may be released this year. I can't say much about it but it's completely different so when everyone has moved to Kirby 3, this plugin will be useless. Therefor I will not spend more time on this. Sorry.

**Comment by Turqueso** [comment]
I agree with Julian, ot's an awesome plugin. @jenstornell But with Kirby 3 close, does it mean this plugin is no longer maintained?

**Comment by jenstornell** [comment]
Yes it will be completely useless in Kirby 3 and it's not possible to port it without a full rewrite. Therefor I'm not maintaing it.
